src/format.iced

```diff
@@ -0,0 +1,114 @@
+# This class prettifies the output of our armoring stream, according to the saltpack spec.
+# At the moment, per the spec, it simply frames the message, and inserts a space every 15 characters
+# and a newline every 200 words.
+
+stream = require('keybase-chunk-stream')
+util = require('./util')
+
+# Punctuation - this is modular
+space = new Buffer(' ')
+newline = new Buffer('\n')
+punctuation = new Buffer('.')
+
+words_per_line = 200
+chars_per_word = 15
+
+# For working with the chunk streams
+noop = () ->
+
+# This stream takes basex'd input and writes framed output, as per https://saltpack.org/armoring#framing-the-basex-payload
+exports.FormatStream = class FormatStream extends stream.ChunkStream
+
+  # _format is the transform function passed to the chunk stream constructor
+  _format : (chunk) ->
+    # this is only ever called during flush()
+    if chunk.length < @block_size
+      return chunk
+
+    # write out the header if we haven't already
+    res = new Buffer('')
+    unless @_header_written
+      res = Buffer.concat([@_header, punctuation, space])
+      @_header_written = true
+
+    @_word_count++
+    if @_word_count isnt 200
+      # res will be an empty buffer except when writing the header
+      res = Buffer.concat([res, chunk, space])
+    else
+      # ditto
+      res = Buffer.concat([res, chunk, newline])
+    return res
+
+  _flush : (cb) ->
+    super(noop)
+    @push(Buffer.concat([punctuation, space, @_footer, punctuation]))
+    cb()
+
+  constructor : ({brand}) ->
+    if brand? then _brand = brand else _brand = 'KEYBASE'
+    @_header = new Buffer("BEGIN#{space}#{_brand}#{space}SALTPACK#{space}ENCRYPTED#{space}MESSAGE")
```

Comment:
As per above, I think parametrizing `#{space}` is more trouble than it's worth, but we might want to parametrize the last two words, in anticipation of signature support.